pori_python/ipr/ipr.py

```diff
@@ -189,40 +191,40 @@ def convert_statements_to_alterations(
         for variant in variants:
             if variant["@rid"] not in variant_matches:
                 continue
+            stmt = {
+                "approvedTherapy": approved_therapy or False,
+                "category": ipr_section or "unknown",
+                "context": (statement["subject"]["displayName"] if statement["subject"] else ""),
+                "kbContextId": (statement["subject"]["@rid"] if statement["subject"] else ""),
+                "disease": ";".join(sorted(d.get("displayName", "") for d in diseases)),
+                "evidenceLevel": evidence_level_str or "",
+                "iprEvidenceLevel": ipr_evidence_levels_str or "",
+                "kbStatementId": statement["@rid"],
+                "matchedCancer": disease_match,
+                "reference": pmid,
+                "relevance": statement["relevance"]["displayName"],
+                "kbRelevanceId": statement["relevance"]["@rid"],
+                "externalSource": (
+                    str(statement["source"].get("displayName", "")) if statement["source"] else ""
+                ),
+                "externalStatementId": statement.get("sourceId", "") or "",
+                "reviewStatus": statement.get("reviewStatus", "") or "",
+                "kbData": {},
+            }
```

Comment:
Just a note about how we could add info about statement's conditional variants.
I think statement conditions are accessible from here and could be added to stmt somehow. Maybe something like:
```
stmt ={
    ...,
    "conditionalVariants": [
        {"@class": cl, "@rid": rid, "displayName": display}
        for (cl, rid, display) in zip(
            statement["conditions.@class"],
            statement["conditions.@rid"],
            statement["conditions.displayName"],
        )
        if cl in VARIANT_CLASSES
    ],
}
```